Cargo.toml

```diff
@@ -149,6 +149,9 @@ bevy_ui = [
   "bevy_color",
 ]
 
+# Enable the bevy_ui debug overlay
+bevy_ui_debug = ["bevy_internal/bevy_ui_debug"]
```

Comment:
IMO we shouldn't expose these features at the `bevy` level at all: instead, we can let users control them at the debug-tools crate level.

Reply:
So, I'll remove it and for now, and we wait for #12368 to be capable of disable this?

Reply:
I would also not add a plugin by default, and let user specifically add it themself.